servers/audio/audio_stream.cpp

```diff
@@ -157,13 +157,15 @@ int AudioStreamPlaybackResampled::mix(AudioFrame *p_buffer, float p_rate_scale,
 			mixed_frames_total = i;
 		}
 
+		AudioFrame y1_tan = y2 - y0;
+		AudioFrame y2_tan = y3 - y1;
 		float mu2 = mu * mu;
-		AudioFrame a0 = 3 * y1 - 3 * y2 + y3 - y0;
-		AudioFrame a1 = 2 * y0 - 5 * y1 + 4 * y2 - y3;
-		AudioFrame a2 = y2 - y0;
-		AudioFrame a3 = 2 * y1;
+		float h11 = mu2 * (mu - 1);
+		float z = mu2 - h11;
+		float h01 = z - h11;
+		float h10 = mu - z;
 
-		p_buffer[i] = (a0 * mu * mu2 + a1 * mu2 + a2 * mu + a3) / 2;
+		p_buffer[i] = y1 + (y2 - y1) * h01 + (y1_tan * h10 + y2_tan * h11) * 0.5;
```

Comment:
`y1_tan`, `y2_tan`, `h01` and `h10` seem to be used a single time in the operation. They could be used directly like:
```
p_buffer[i] = y1 + (y2 - y1) * (z - h11) + ((y2 - y0) * (mu - z) + (y3 - y1) * h11) * 0.5;
```

Just a suggestion.

Reply:
I can do this for `y1_tan` and `y2_tan`, but I feel like for `h10` and `h01` it would "un-self-document" the code, so to speak.